pkg/controller.v1/common/job.go

```diff
@@ -67,6 +68,26 @@ func (jc *JobController) cleanupJobIfTTL(runPolicy *apiv1.RunPolicy, jobStatus a
 	return nil
 }
 
+
+// recordAbnormalPods records the active pod whose latest condition is not in True status.
```

Comment:
I think you meant recording the condition of the pod?

Reply:
yeah